### DISCUSSION TITLE
Understanding why headers are missing in http.proto

### DISCUSSION THREAD

**Discussion by ikouchiha47** [googleapis/googleapis#749] [Q&A]
Hi.
So we are using this grpc-gateway thing. And mostly for request response struct generation and doc generation.
I was stuck at annotating arbitrary request headers. 
In [http.proto](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) i don't see anything that represents request headers.

So we wanted to understand why is that and if there are any alternative ways or proto definition like http.proto that will help with this.

In case it's desirable we can work on sending a PR to the required repositories

**Comment by toumorokoshi** [comment]
+1 it would be great to have this feature - we would like it for aep.dev to enable gRPC-HTTP APIs that are compatible with IETF RFCs.

I've started a PR here to start the conversation: https://github.com/googleapis/googleapis/pull/1005
